docs/graph/getting-started.asciidoc

```diff
@@ -1,57 +1,62 @@
 
 [[exploring-connections]]
 To start exploring connections in your data:
 
-. Open Kibana in your web browser and log in. If you are running Kibana
-locally, go to `http://localhost:5601/`.
-
-. Click **Graph** in the side navigation to open the graph explorer.
-+
-image::graph/images/graph-open.jpg["Accessing Graph"]
+. From the side navigation, open the graph explorer.
 
 . Select an index pattern to specify what indices you want to explore.
++
 For example, if you are indexing log data with Logstash, you could select the
 `logstash-*` index pattern to visualize connections within the log entries.
 
 . Select one or more multi-value fields that contain the terms you want to
-graph. The vertices in the graph are selected from these terms. If you're
+graph. 
++
+The vertices in the graph are selected from these terms. If you're
 visualizing connections between Apache log entries, you could select the
 `url.raw` field and the `geo.src` field so you can look at which pages are
 being accessed from different locations.
 
 . Enter a search query to discover relationships between terms in the selected
-fields. For example, to generate a graph of the successful requests to
+fields. 
++
+For example, to generate a graph of the successful requests to
 particular pages from different locations, you could search for the 200
-response code:
+response code. The weight of the connection between two vertices indicates how strongly they
+are related. 
 +
-image::graph/images/graph-url-connections.jpg["URL connections"]
-
-The weight of the connection between two vertices indicates how strongly they
-are related. You can click any connection to view more information about
-the relationship:
-
-image::graph/images/graph-link-summary.jpg["Link summary"]
-
-Once you have your initial graph, you can use the toolbar buttons to explore
-additional connections. Click the Expand button
-image:graph/images/graph-expand-button.jpg[Expand Selection] to display additional vertices
-that connect to your graph. Click the Link button
-image:graph/images/graph-link-button.jpg[Add links to existing terms] to display additional
-connections between the displayed vertices. To explore a particular area of the
-graph, select the vertices you are interested in and click the Expand or Link button.
-To step back through your changes to the graph, click the Undo button
-image:graph/images/graph-undo-button.jpg[Undo].
+[role="screenshot"]
+image::graph/images/graph-url-connections.png["URL connections"]
 
-To see more relationships within your data, you can submit additional queries.
+. To view more information about the relationship, click any connection.
++
+[role="screenshot"]
+image::graph/images/graph-link-summary.png["Link summary"]
 
-image::graph/images/graph-add-query.jpg["Adding networks"]
+. Use the toolbar buttons to explore
+additional connections:
++
+* To display additional vertices that connect to your graph, click the Expand 
```

Comment:
Maybe I'm misreading the syntax, but shouldn't this be `click Expand` or `click the Expand button` ?